muxdb/engine/leveldb.go

```diff
@@ -112,14 +114,20 @@ func (ldb *levelEngine) Bulk() kv.Bulk {
 		return batch
 	}
 	flush := func(minSize int) error {
-		if batch != nil && len(batch.Dump()) >= minSize {
-			if batch.Len() > 0 {
-				if err := ldb.db.Write(batch, &writeOpt); err != nil {
-					return err
+		if batch != nil {
+			if batchBytesNo := len(batch.Dump()); batchBytesNo >= minSize {
+				if batch.Len() > 0 {
+					startTime := mclock.Now()
+					if err := ldb.db.Write(batch, &writeOpt); err != nil {
+						return err
+					}
+					metricBatchWriteBytes().Set(int64(batchBytesNo))
+					batchWriteElapsed := mclock.Now() - startTime
```

Comment:
Hey, does it make sense to change these lines order?
```suggestion
batchWriteElapsed := mclock.Now() - startTime
metricBatchWriteBytes().Set(int64(batchBytesNo))
```

Reply:
They both get different values (`batchBytesNo` for the number of bytes and `batchWriteElapsed` for the time) so IMO it does not make a difference since the event has finished in any case.

But anyways, I am going to accept your suggestion. Thanks for reviewing!